### DISCUSSION TITLE
Option to exclude specific media directories

### DISCUSSION THREAD

**Discussion by voronind** [Archmonger/django-dbbackup#576] [Feature Requests]
It would be useful to skip some media directories from backuping.
E.g. Cache directories

**Comment by benjaoming** [comment]
That's also a great idea! :)

**Comment by ZuluPro** [comment]
We need before to list all media files via Django built-in API.
And after there is `fnmatch` (https://docs.python.org/2/library/fnmatch.html#fnmatch.fnmatch) for match file with wildcard.

We must ensure storages are able to list all files in all directories, files' names must be an absolute path from storage's root. Maybe add a `walk` function to storages.

**Comment by ZuluPro** [comment]
Now DBbackup can walk through storage, excludes are coming soon !

**Comment by jonathan-s** [comment]
covered in #300
